components/pagination/src/Pagination.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { FC, createElement as h } from 'react';
+import { StandardProps, classBuilder } from '@not-govuk/component-helpers';
+
+import '../assets/Pagination.scss';
+
+export type PaginationProps = StandardProps & {
+  results: number
+  resultsPerPage: number
+  page: number
+  previous?: string
+  pages?: number[]
+  next?: string
+};
+
+export const Pagination: FC<PaginationProps> = ({
+  children,
+  classBlock,
+  classModifiers,
+  className,
+  results,
+  resultsPerPage,
+  previous,
+  page,
+  pages,
+  next,
+  ...attrs
+}) => {
+  const classes = classBuilder(
+    'hods-pagination',
+    classBlock,
+    classModifiers,
+    className
+  );
+
+  var resultsFrom: number = ((page-1)*resultsPerPage)+1
+  var resultsTo: number = resultsFrom + (resultsPerPage - 1)
+  {resultsTo>results ? resultsTo=results : null}
+
+  return (
+    <div className={classes()}>
+      <div className={classes('summary')}>Showing {resultsFrom} - {resultsTo} of {results} results</div>
+      <ul className={classes('list-items')}>
+        <li className={classes('item')} id="prevButton">
+          <a className={classes('link')} href="#2">
```

Comment:
I'm guessing this comes from the original component, but what is `#2`?

I'm worried that this isn't really defined as a component. i.e. It's more of a pattern. (Any thoughts @karypun?)

Reply:
Possibly we should be doing something like:
```
href="?page=${page}"
```
(Where `page` is an integer we have calculated.)

Reply:
have updated this now